modules/tinymce/CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Color picker dialog now starts on the appropriate color for the cursor position. #TINY-9213
 
 ### Fixed
+- Clicking on list toolbar button inside a table cell that has anchor element when the cursor is outside the anchor element would not include the anchor element in the produced list. #TINY-6853
```

Comment:
Not sure if it's any better writing changelog items is hard. Just noticed that the one you have is unusually long.

```suggestion
- Creating a list in a table cell when the caret is in front of an anchor element would not properly include the anchor in the list. #TINY-6853
```